code/modules/mob/living/brain/brain_item.dm

```diff
@@ -80,8 +80,6 @@
 	name = "[L.name]'s brain"
 	if(brainmob || decoy_override)
 		return
-	if(!L.mind)
-		return
```

Comment:
Does this really interfere with mmis transfering and such? Seems unrelated

Reply:
Robotize transfers the mob's mind into the cyborg before transferring identity to avoid the message that shows when you get debrained

Since mind.transfer_to() nulls the mind of the mob, it would have aborted the creation of the brainmob which we need to transfer species, DNA and stuff